letsencrypt/revoker.py

```diff
@@ -288,7 +288,7 @@ def _remove_certs_from_list(self, cert_list):  # pylint: disable=no-self-use
             :class:`letsencrypt.revoker.Cert`
 
         """
-        list_path2 = tempfile.mktemp(".tmp", "LIST")
+        _, list_path2 = tempfile.mkstemp(".tmp", "LIST")
```

Comment:
Rather than `_` let's turn it into variable. Then we can use it later instead of `with open(list_path2, "wb")`... unless we require binary mode and it's not possible to do it so with `mkstemp` - in this case, we should close the file opened by `mkstemp` not to cause unnecessary leaks.